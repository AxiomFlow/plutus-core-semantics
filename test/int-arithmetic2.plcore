(builtin equalsInt ((builtin subtractInt (40 30)) (builtin addInt (5 5))))
