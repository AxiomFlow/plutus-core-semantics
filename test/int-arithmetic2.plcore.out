<T> <k> true </k> </T>
